public/app/plugins/datasource/grafana-azure-monitor-datasource/components/QueryEditor/QueryEditor.tsx

```diff
@@ -133,6 +130,45 @@ const EditorForQueryType: React.FC<EditorForQueryTypeProps> = ({
         />
       );
 
+    /** Remove with Grafana 9 */
+    case DeprecatedAzureQueryType.ApplicationInsights:
+      if (gtGrafana9()) {
+        return (
+          <Alert title="Deprecated">
+            Application Insights has been deprecated. Use the Metrics service instead.{' '}
+            <a
+              href="https://grafana.com/docs/grafana/latest/datasources/azuremonitor/deprecated-application-insights/#application-insights"
+              target="_blank"
+              rel="noreferrer"
+            >
+              More info
```

Comment:
rather than wrapping the "More info" text here, can we instead wrap "Use the Metrics service" instead? And same for insights analytics?

https://usability.yale.edu/web-accessibility/articles/links#link-text